### DISCUSSION TITLE
Zoom functionality

### DISCUSSION THREAD

**Discussion by Melynt3** [capawesome-team/capacitor-mlkit#71] [Ideas]
Hi,

I asked about this a few days ago in a different topic, but I would like to officially request a new feature, which is zoom functionality. It's a big challenge to work with small barcodes. I encountered a problem when scanning small CODE128 codes; basically, the scanner does not read them. I was able to hardcode a workaround in the library itself by adding a zoom of 1.6f during camera initialization, but this is only a temporary solution. It would be a great feature that would make the library even more attractive.

Cheers o/

**Comment by robingenz** [comment]
Hi @Melynt3,
you have only created a discussion, not a feature request. You can create a feature request here: https://github.com/capawesome-team/capacitor-mlkit/issues/new/choose
I will close this discussion.

**Reply by Melynt3** [reply]
Ok, sorry, im pretty new to forum stuff
